app/views/govuk_publishing_components/component_guide/index.html.erb

```diff
@@ -8,6 +8,7 @@
     <li><a href="https://github.com/alphagov/govuk_publishing_components/blob/master/docs/component_conventions.md">follow component conventions</a></li>
   </ul>
   <p>See the <a href="https://github.com/alphagov/govuk_publishing_components">govuk_publishing_components gem</a> for further details, or <a href="https://docs.publishing.service.gov.uk/manual/components.html#component-guides">a list of all component guides</a>.</p>
+  <p>If you cannot find a component to fit your need, ask in #govuk-frontenders. If there is no suitable component, please use the <a href="https://design-system.service.gov.uk/styles/">Design System styles</a>.</p>
```

Comment:
I think we should give devs a link to the general Design System docs as well.

Reply:
I'm concerned that with '_if there is no suitable component, please use the <a href="https://design-system.service.gov.uk/styles/">Design System styles</a>_' we will promote class composition instead of isolation and re-usability.